packages/suite/src/components/wallet/Pagination.tsx

```diff
@@ -98,17 +100,19 @@ export const Pagination = ({
             </Actions>
 
             {totalPages ? (
-                calculatedPages.map(i => (
-                    <PageItem
-                        key={i}
-                        data-testid={`@wallet/accounts/pagination/${i}`}
-                        data-test-activated={i === currentPage}
-                        onClick={() => onPageSelected(i)}
-                        $isActive={i === currentPage}
-                    >
-                        {i}
-                    </PageItem>
-                ))
+                calculatedPages
+                    .slice(0, limitedPageList ? currentPage + 2 : calculatedPages.length)
```

Comment:
Why +2?

Reply:
2 more pages visible, aswe agreed. (**1**, 2, 3) not just (**1**, 2)

Reply:
maybe we could add it to some const to make it more clear, but nitpick

Reply:
okay